Data models

josemontesdeoca opened the pull request:
Add initial data models with Parse support.

/cc @fcduarte

fcduarte commented on src/org/missionassetfund/apps/android/MAFApplication.java:
```diff
@@ -1,19 +1,29 @@
         // Initializing Parse
-        Parse.initialize(this, "YeCq7QZpwMg4vEqaZlJs2ZVZvQ7gJrTi1zN9aa4t",
-                "DgHFqxILVyV2qIGA0pLNrbMlNMQBqDIpI5PZjV9x");
+        Parse.initialize(this, "qK7qJuFt6weBIrBx9eTzK1UBWJvkqb3jH6l8aw22",
```
Tonight we could discuss with @nesquena the best way to do it since this is a public repo, maybe we could externalize to a XML file... What do you think?

fcduarte commented:
Reviewed. Just added a note for remembering to talk about it.

josemontesdeoca committed:
Protect Parse app/client id

josemontesdeoca committed:
Model initial schema to Parse Objects

josemontesdeoca commented:
@fcduarte I've cleaned up this PR. removed Parse app/client id and added an app_config.xml file to read this info which is not source controlled. Let me know your thoughts

fcduarte commented:
Great :ship:
